services/QuillLMS/app/views/application/_head.html.erb

```diff
@@ -38,7 +38,7 @@
   <meta name="og:description" content="<%= @description || @content %>">
   <link rel="apple-touch-icon" href="<%= ENV['CDN_URL'].gsub(/[‘’]/, '') + "/images/logos/apple-touch-icon.png" %>">
   <link rel="canonical" href="https://www.quill.org<%= request.path %>">
-  <link rel="manifest" href="manifest.json" />
+  <link rel="manifest" href="<%= ENV['DEFAULT_URL'] + "/manifest.json" %>" />
```

Comment:
I think there's a slightly cleaner way of doing this that relies on the subtleties of `href`.  `filename.json` will build a link to a path that's the same as the current page with only the section after the last `/` changed.  `host/session/new` would become `host/session/filename.json`, which is what I assume you have been running into here.  But if you specify a `href` value starting with `/`, it creates a link to a path that's just the host and the `href` value.  So `quill.org/session/new` gets changed to `quill.org/filename.json`.

All of which is a long-winded way of saying that I think that instead of including `ENV['DEFAULT_URL']`, you can just make this `href="/manifest.json"`